Enable TinyTeX to be installed relative to the conda prefix in which Quarto is installed

diff --git a/src/tools/impl/tinytex-info.ts b/src/tools/impl/tinytex-info.ts
index 34c10a0..47dc25d 100644
--- a/src/tools/impl/tinytex-info.ts
+++ b/src/tools/impl/tinytex-info.ts
@@ -21,6 +21,16 @@ export function hasTinyTex(): boolean {
 }
 
 export function tinyTexInstallDir(): string | undefined {
+  const condaPrefix = getenv("QUARTO_CONDA_PREFIX");
+  if (condaPrefix) {
+    switch (Deno.build.os) {
+      case "windows":
+        return join(condaPrefix, "Library");
+      default:
+        return condaPrefix;
+    }
+  }
+
   switch (Deno.build.os) {
     case "windows":
       return expandPath(join(getenv("APPDATA", undefined), "TinyTeX"));
@@ -34,6 +44,11 @@ export function tinyTexInstallDir(): string | undefined {
 }
 
 export function tinyTexBinDir(): string | undefined {
+  const condaPrefix = getenv("QUARTO_CONDA_PREFIX");
+  if (condaPrefix) {
+    return join(condaPrefix, "bin");
+  }
+
   const basePath = tinyTexInstallDir();
   if (basePath) {
     switch (Deno.build.os) {
